source/_integrations/ping.markdown

```diff
@@ -86,14 +86,15 @@ To use this presence detection in your installation, add the following to your `
 device_tracker:
   - platform: ping
     hosts:
-      hostname: 192.168.2.10
+      hostname1: 192.168.2.10
+      hostname2: 192.168.2.11
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters that they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

```suggestion
```

Reply:
Reading the docs again is says this is a map of device name to hostname or IP address. So maybe instead we make it:

```
hosts:
    device_name_1: 192.168.2.10
```

What do you think?

Reply:
Updated and added one more commit to fix the link to the people-tracking documentation.